browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -1064,6 +1068,41 @@ class TreeViewControl {
 		}, 100);
 	}
 
+	highlightEntries(searchTerm: string) {
+		app.highlightTerm = searchTerm;
```

Comment:
Not sure if its ok to use `app` to store what we are currently searching. Seems like it could be bad practice to store such a small bit of information globally. 

Any alternative suggestions are appreciated.